visualization/MANUAL_TESTING.md

```diff
@@ -354,21 +354,6 @@
 
 ## When
 
--
```

Comment:
This document was hard work but we never used it in the last 2 years. Should we remove this?
Tests should be automated.

Reply:
I had a brief look at it and it does seem rather outdated. I agree that it's probably best to remove it.